src/Namotion.Reflection/XmlDocsExtensions.cs

```diff
@@ -707,11 +707,15 @@ internal static string GetMemberElementName(dynamic member)
                     {
                         path = DynamicApis.PathCombine(baseDirectory, assemblyName.Name + ".xml");
                         if (DynamicApis.FileExists(path))
+                        {
+                            return path;
+                        }
+
+                        path = DynamicApis.PathCombine(baseDirectory, "bin\\" + assemblyName.Name + ".xml");

```

Comment:
We should use bin/ instead of bin\ to also support Unix systems...

Reply:
You true. Fixed that.